classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
+    student.classroom = self
+  end
+end
```

Comment:
- [ ] For accessing the classroom, there should be an attr_reader for a classroom. Kindly add that. `attr_reader :classroom` or better you could use `attr_accessor :label, :classroom` instead 👍🏽 as this does both getter and setter functionalities.

Reply:
Hello @AmaduKamara , thank you for the feedback. I don't understand why the Classroom class should have a getter/setter for :classroom, could you kindly clarify, please? I did what you describe in the Student class since a Student belongs to a Classroom, but I'm not sure I understand why this is necessary for the Classroom class. Thank you very much in advance for your time

Reply:
Hi, sorry bro. 
It was a mistake, my bad, apologies please.

Reply:
Sorry for late response. 
Thank you and good luck 👍